requirements/edx/github.txt

```diff
@@ -68,6 +68,8 @@ git+https://github.com/hmarr/django-debug-toolbar-mongo.git@b0686a76f1ce3532088c
 git+https://github.com/edx/rfc6266.git@v0.0.5-edx#egg=rfc6266==0.0.5-edx
 # Used for testing
 git+https://github.com/edx/lettuce.git@0.2.20.002#egg=lettuce==0.2.20.002
+# OpenStack swift backend for django storage API
+git+https://github.com/open-craft/django-storage-swift.git@eb812cb34a266a58558de4d05ff08b3afd025e36#egg=django-storage-swift==1.2.7
```

Comment:
Done in a833abd. See also: https://github.com/open-craft/configuration/commit/e1909ca727f82091b258e6b9529bb1516d723872